src/windows/2004/mono/amd64/Dockerfile

```diff
@@ -0,0 +1,76 @@
+# escape=`
+FROM mcr.microsoft.com/windows:2004
+
+USER ContainerAdministrator
+
+# Download and install cmake
+RUN curl -SL --output %TEMP%\cmake-x64.msi https://cmake.org/files/v3.9/cmake-3.9.6-win64-x64.msi `
+    && powershell -Command `
+        Start-Process %TEMP%\cmake-x64.msi -Wait -ArgumentList '/qn /norestart' ; `
+        Remove-Item -Recurse -Force %TEMP%\*
+
+# Download and install python
+RUN curl -SL --output %TEMP%\python2-x64.msi https://www.python.org/ftp/python/2.7.18/python-2.7.18.amd64.msi `
+    && powershell -Command `
+        Start-Process %TEMP%\python2-x64.msi -Wait -ArgumentList '/qn /norestart TARGETDIR=C:\python' ; `
+        Remove-Item -Recurse -Force %TEMP%\*
+
+# Download and install Mono
+RUN curl -SL --output %TEMP%\mono32.msi https://download.mono-project.com/archive/6.12.0/windows-installer/mono-6.12.0.107-gtksharp-2.12.45-win32-0.msi `

```

Comment:
we can remove this I think, it was used before we had a working monolite which is downloaded during the build. I've never actually seen it being used since we used to install an older Mono that wasn't compatible anyway.

Reply:
I think this was to provide `mcs` in absence of Monolite, back in the day